entities.py

```diff
@@ -0,0 +1,118 @@
+from dices import Dices
+import random
+
+
+class Movement():
+    def __init__(self):
```

Comment:
This init has a small problem, as it will check the move options when it is called upon, but we can't change the position before we call it,. Because of that, it might be smart to remove the check_options call from the init.